secretsd/database.py

```diff
@@ -39,7 +41,7 @@ def get_next_object_id(self):
     # Collections
 
     def add_collection(self, object, label):
-        print("DB: adding collection %r with props %r" % (object, props))
+        print("DB: adding collection %r with label %r" % (object, label))
```

Comment:
Seems like this was a leftover from earlier code. Was causing a python exception, preventing the default collection from being created.